test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -146,42 +149,55 @@ public void getParticipants() throws Exception {
         assertEquals(new Integer(10), page.getOffsetBy());
         assertEquals(20, page.getPageSize());
         assertEquals("foo", page.getFilters().get("emailFilter"));
-        verify(participantService).getPagedAccountSummaries(STUDY, 10, 20, "foo");
+        verify(participantService).getPagedAccountSummaries(study, 10, 20, "foo");
     }
     
     @Test(expected = BadRequestException.class)
     public void oddParametersUseDefaults() throws Exception {
         controller.getParticipants("asdf", "qwer", null);
         
         // paging with defaults
-        verify(participantService).getPagedAccountSummaries(STUDY, 0, API_DEFAULT_PAGE_SIZE, null);
+        verify(participantService).getPagedAccountSummaries(study, 0, API_DEFAULT_PAGE_SIZE, null);
     }
 
     @Test
     public void getParticipant() throws Exception {
-        StudyParticipant studyParticipant = new StudyParticipant.Builder().withFirstName("Test").build();
-        
-        when(participantService.getParticipant(STUDY, CALLER_ROLES, ID)).thenReturn(studyParticipant);
+        study.setHealthCodeExportEnabled(true);
+        StudyParticipant studyParticipant = new StudyParticipant.Builder().withFirstName("Test").withHealthCode("healthCode").build();
+        when(participantService.getParticipant(study, ID, true)).thenReturn(studyParticipant);
         
         Result result = controller.getParticipant(ID);
-        String string = Helpers.contentAsString(result);
-        StudyParticipant retrievedParticipant = BridgeObjectMapper.get().readValue(string, StudyParticipant.class);
-        // Verify that there's a field, full serialization tested in StudyParticipant2Test
+        String json = Helpers.contentAsString(result);
+        StudyParticipant retrievedParticipant = BridgeObjectMapper.get().readValue(json, StudyParticipant.class);
+        
         assertEquals("Test", retrievedParticipant.getFirstName());
+        assertNull("healthCode", retrievedParticipant.getHealthCode()); // for now This will eventually return health code.
+    }
+    
+    @Test
+    public void getParticipantWithNoHealthCode() throws Exception {
+        study.setHealthCodeExportEnabled(false);
+        StudyParticipant studyParticipant = new StudyParticipant.Builder().withFirstName("Test").withHealthCode("healthCode").build();
+        when(participantService.getParticipant(study, ID, true)).thenReturn(studyParticipant);
         
-        verify(participantService).getParticipant(STUDY, CALLER_ROLES, ID);
+        Result result = controller.getParticipant(ID);
+        String json = Helpers.contentAsString(result);
+        StudyParticipant retrievedParticipant = BridgeObjectMapper.get().readValue(json, StudyParticipant.class);
+        
+        assertEquals("Test", retrievedParticipant.getFirstName());
+        assertNull(retrievedParticipant.getHealthCode());
```

Comment:
Similarly, do we want to explicitly test both getHealthCode() and getEncryptedHealthCode()?